lambdaclass/mirra_backend
# Gh 894 power ups

## Motivation

Redesign the way we acquire power ups through the game.

There's an existing bug where you get extra power up from crates if you hit it with two projectiles that will be fixed in https://github.com/lambdaclass/mirra_backend/pull/904
  
Closes #894 

## Summary of changes

- [feat: add new feature to give power ups to the killer](https://github.com/lambdaclass/mirra_backend/pull/899/commits/acb3b75d60a878ec8e99b8b7848cd0208cdfaeca)
- [refactor: remove no longer used variable](https://github.com/lambdaclass/mirra_backend/pull/899/commits/1c9215866763cf988802ec64a656b0756604dfc4)
- [feat: add new feature to give power ups dropped by crates to the player](https://github.com/lambdaclass/mirra_backend/pull/899/commits/f8db95c53bc5ef0dc7b2ef8f2c40130b08bbbfdc)
- Fix effects dealing damage to dead entities

## How to test it?

Test the game in [this branch](https://github.com/lambdaclass/champions_of_mirra/pull/2093) and instead of the crates  dropping the power ups to the ground you should see your power up number go up

## Checklist
